services/web/client/source/class/qxapp/data/Permissions.js

```diff
@@ -97,7 +97,8 @@ qx.Class.define("qxapp.data.Permissions", {
     getLowerRoles(role) {
       const lowerRoles = [];
       for (const r in this.self().ROLES) {
-        if (r === role) {
+        if (r.toLowerCase() === role.toLowerCase()) {
+          lowerRoles.push(r);
```

Comment:
getLowerRoles(role) : This function returns you all roles until it matches the one you have? Aha ... by "lower" you mean "children" or "below" ...  
What if you have multiple inheritance? is the ROLES built in a special way?
Shouln't you use ``inherits`` instead?